x/bank/keeper.go

```diff
@@ -24,69 +29,48 @@ type Keeper interface {
 	AddCoins(ctx sdk.Context, addr sdk.AccAddress, amt sdk.Coins) (sdk.Coins, sdk.Tags, sdk.Error)
 }
 
-var _ Keeper = (*BaseKeeper)(nil)
-
 // BaseKeeper manages transfers between accounts. It implements the Keeper
 // interface.
 type BaseKeeper struct {
-	am auth.AccountKeeper
-}
+	BaseSendKeeper
 
-// NewBaseKeeper returns a new BaseKeeper
-func NewBaseKeeper(am auth.AccountKeeper) BaseKeeper {
-	return BaseKeeper{am: am}
+	ak auth.AccountKeeper
 }
 
-// GetCoins returns the coins at the addr.
-func (keeper BaseKeeper) GetCoins(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins {
-	return getCoins(ctx, keeper.am, addr)
+// NewBaseKeeper returns a new BaseKeeper
+func NewBaseKeeper(ak auth.AccountKeeper) BaseKeeper {
+	return BaseKeeper{BaseSendKeeper: NewBaseSendKeeper(ak), ak: ak}
```

Comment:
should use separate lines for fields struct object declaration

Reply:
For sure!